src/main/views/remove-list-search-results.njk

```diff
@@ -38,6 +39,7 @@
                         {% elif (item.language == 'BI_LINGUAL') %}
                           <td class="govuk-table__cell">{{ languageItems[2] }}</td>
                         {% endif %}
+                      <td class="govuk-table__cell">{{ item.sensitivity | titleCase }}</td>
```

Comment:
Something wrong on the indentation on this file. Are you ok to fix that while you are there?

Reply:
Absolutely. Sorted now :)